charts/posthog/Chart.yaml

```diff
@@ -11,7 +11,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 30.5.1
+version: 30.5.2
```

Comment:
The annoying thing about this is that we are not backwards compatible with older docker images. In this case I _think_ we'll require all dependencies. So would make sense to ensure the upgrade is not rolled out automatically, maybe a minor or major version.

I haven't found a great way to handle this, aside from maybe removing the `role` abstraction and rather allowing specifically the dependencies that should be checked.

I've not got a good answer here 🤷

Reply:
That decide pool is not enabled by default, and requires two values to be set before receiving traffic, so I think it's fine to move forward.
We can add a comment to the `decide.enabled` value that it requires posthog 1.44.0 (we'll do this one before switching to continuous releases, right?)

Reply:
Hah yeah forgot it was behind a flag. Yeah adding a note to the release doc sounds like a plan 👍